static/src/javascripts/projects/common/modules/identity/upsell/consent-card/FollowCardList.js

```diff
@@ -117,6 +118,17 @@ class FollowCardList extends Component<
                                 less: 'Less',
                             }}
                         />
+                        {isExpanded && (
+                            <div>
+                                <a
+                                    className="u-underline identity-upsell-consent-card__link"
+                                    href={`${config.get(
+                                        'page.host'
+                                    )}/email-newsletters`}>
```

Comment:
Yeah I don't think it's great either, I didn't actually know this page even existed. When I asked last week, Katya told me to link to this page, I guess because you don't need to set a password for it, but she didn't seem keen either. Will see what she think tomorrow about linking to email-prefs